### ISSUE TITLE
ScheduledStop bug fixes

### ISSUE THREAD

**Issue by waicool20** [mrmin123/kancolle-auto#329]
Some copy pasta derps in this section:

```            
if settings['scheduled_stop_mode'] == 'time':
                now_time = datetime.datetime.now()
                if now_time > settings['scheduled_stop_time']:
                    log_success("kancolle-auto has ran for the desired %s hours! Shutting down now!" % settings['scheduled_stop_count'])
                    stop_flag = True
            elif settings['scheduled_stop_mode'] == 'expedition':
                if done_expeditions >= settings['scheduled_stop_count']:
                    log_success("kancolle-auto has ran the desired %s expeditions! Shutting down now!" % settings['scheduled_stop_count'])
                    stop_flag = True
            elif settings['scheduled_stop_mode'] == 'sortie':
                if done_expeditions >= settings['scheduled_stop_count']:
                    log_success("kancolle-auto has ran for the desired %s sorties! Shutting down now!" % settings['scheduled_stop_count'])
                    stop_flag = True
            elif settings['scheduled_stop_mode'] == 'pvp':
                if done_expeditions >= settings['scheduled_stop_count']:
                    log_success("kancolle-auto has ran for the desired %s pvps! Shutting down now!" % settings['scheduled_stop_count'])
                    stop_flag = True
```

```done_expeditions``` for ```expedition```, ```sortie``` and ```pvp``` mode?

**Comment by waicool20** [comment]
Also should be noted that since you put ```kancolle_auto_wrapper()``` in a ```while True:``` loop, even when it stops via ScheduledStop it just gets rebooted immediately so a ```sys.exit(0)``` needs to be put in

**Comment by waicool20** [comment]
And also when ```ScheduledStopCount``` and mode is set to something other than ```time``` it only stops when the ```done_<mode>``` is ```settings['scheduled_stop_count'] + 1```

So eg. I set it to stop after 1 sortie it only does so after 2 sorties

**Comment by mrmin123** [comment]
Merged #330. Thanks for catching and fixing this!
